InternetShop/src/main/java/model/Order.java

```diff
@@ -1,34 +1,36 @@
 package model;
 
 import java.math.BigDecimal;
-import java.util.ArrayList;
 
 import java.util.List;
+import java.util.Random;
 
 public class Order {
-
     private long id;
     private List<PositionItem> itemList;
     private BigDecimal total;
-    private User user;
+    private String user;
     private StatusOrder statusOrder;
 
     public enum StatusOrder{
         COMPLITED,IN_PROCESS,REFUSAL
     }
 
-    public Order(List<PositionItem> itemList, User user, StatusOrder statusOrder) {
+    public Order(){}
+    public Order(List<PositionItem> itemList, String user, StatusOrder statusOrder) {
         this.itemList = itemList;
         this.user = user;
         this.statusOrder = statusOrder;
+        this.total = getTotal();
+        this.id = new Random().nextInt(999999999);
```

Comment:
its not good to use Random for Id generation, it may cause same ids -> data will be overwritten.
its better  to have
 private static long Idgenerator;
and then
this.id = ++Idgenerator;